### DISCUSSION TITLE
Custom rules syntax help

### DISCUSSION THREAD

**Discussion by morelgeorge** [NatVanG/PBIXInspector#32] [Q&A] [answered]
Hello, 
I have a question regarding syntax of the rules. In the documentation it is described that rules are using JsonLogic syntax. So I followed the documentation. However, in provided base-rules.json file there are functions/operators that are not described in provided documentation file (e.g. using count, drillvar, strcontains, etc.). 

Is there any document where I can find really all the possible functions we can use for writing custom rules? 

Thanks a lot!

**Comment by morelgeorge** [comment, marked as answer]
OK, I missed this wiki page - https://github.com/NatVanG/PBI-Inspector/wiki/Operators 
It is become to be more clear now.

**Reply by NatVanG** [reply]
Sorry I had meant to reply sooner. Glad you found that wiki page. I aim to add more documentation asap.
